bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/RetryingReadRowsOperation.java

```diff
@@ -148,15 +148,13 @@ protected ReadRowsRequest getRetryRequest() {
   public void run() {
     try {
       // restart the clock.
-      synchronized (callLock) {
-        super.run();
-        // pre-fetch one more result, for performance reasons.
-        adapter.request(1);
-        if (rowObserver instanceof ClientResponseObserver) {
-          ((ClientResponseObserver<ReadRowsRequest, FlatRow>) rowObserver).beforeStart(adapter);
-        }
-        lastResponseMs = clock.currentTimeMillis();
+      super.run();
```

Comment:
is synchronized (callLock) not required anymore?

Reply:
Correct.  It's not needed anymore.  There were some strange cases where call was null, but it isn't ever null anymore with this change.